### PR TITLE
[15.0][MIG] stock_warehouse_flow

### PR DESCRIPTION
**Pull request by sebalix**

### CODE REVIEW

**Comment by sebalix** [comment]
cc @jbaudoux

File: stock_warehouse_flow/tests/common.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields
+from odoo.tests.common import SavepointCase
```
Comment (rousseldenis) [inline stock_warehouse_flow/tests/common.py]: @sebalix This is deprecated.

Comment (sebalix) [reply stock_warehouse_flow/tests/common.py]: fixed

**Pull request closed without merging**